e2e-tests/ukef/cypress/support/commands.js

```diff
@@ -48,3 +47,9 @@ Cypress.Commands.add('forceVisit', (url) => {
 
 Cypress.Commands.add('submitDeal', require('./trade-finance-manager-api/submitDeal'));
 Cypress.Commands.add('submitManyDeals', require('./trade-finance-manager-api/submitManyDeals'));
+
+// shared commands for TFM
+Cypress.Commands.add('login', require('../../../support/trade-finance-manager/login'));
```

Comment:
We have 2 login commands in `e2e-tests/ukef`  , second command could have alias `tfmLogin`, like it had before.